### DISCUSSION TITLE
Friday night funkin kade engine 1.6.1 how to change each position of campaign_menu_UI_assets

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1643] [Q&A] [answered]
Please tell me how to change each position of campaign_menu_UI_assets

**Comment by Riskerman21** [comment, marked as answer]
In storymenustate.hx:

                 var ui_tex = Paths.getSparrowAtlas('campaign_menu_UI_assets');
		var yellowBG:FlxSprite = new FlxSprite(0, 56).makeGraphic(FlxG.width, 400, 0xFFF9CF51);



Change the 0, 56 to the number you want.

**Reply by Reyororo** [reply]
thank you
